packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -15,10 +15,12 @@ import SearchIcon from 'coveo-styleguide/resources/icons/svg/search.svg';
 export interface AtomicSearchBoxOptions {
```

Comment:
I think we can remove this interface, document the props directly, and their default should be picked up by Stencil correctly